staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/internal/default_pruning.go

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
is this a copy of the algorithm used by builder? Can we merge them?

Reply:
not really, the builder goes from CRD type representation to `swagger.Spec`. This is operating directly on the swagger`.Spec`. On the other hand, I don't think there is any call left to the builder with StripDefaults to true, so we might be able to remove it.

Reply:
OK I just removed it!